src/web/modules/gateway_accounts/email_branding.njk

```diff
@@ -0,0 +1,42 @@
+{% from "components/input/macro.njk" import govukInput %}
+{% from "components/button/macro.njk" import govukButton %}
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <h1 class="govuk-heading-m">Update email custom branding</h1>
```

Comment:
Very minor comment but I quite like the caption + heading as used on some of the newer 'update' pages. 

Something like 
https://github.com/alphagov/pay-toolbox/blob/5280ebb0deee659e756e7817986fadc07ea88869/src/web/modules/users/users.update_email.njk#L7-L8

Might be `Update gateway account` -> `Email custom branding`

Reply:
👍 nice, I've changed it